docs/resources/vba-differences.md

```diff
@@ -16,7 +16,7 @@ This article describes the main differences between VBA macros (as well as VBA i
 
 ## Platform and ecosystem
 
-VBA is designed for the desktop and Office Scripts are designed for the web. VBA can interact with a user's desktop. This lets it integrate with similar technologies, such as COM and OLE. However, VBA has no convenient way to call out to the internet.
+VBA is designed for the desktop and Office Scripts are designed for the web. VBA can interact with a user's desktop. This lets it work with similar technologies, such as COM and OLE. However, VBA has no convenient way to call out to the internet.
```

Comment:
I feel like "lets it work" is weird phrasing? I don't have a great alternative suggestion but maybe look at the previous sentence and see if there's some kind of combining that can happen between them